[2024-01-28T13:42:38.356+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-15T00:00:00+00:00 [queued]>
[2024-01-28T13:42:38.426+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-15T00:00:00+00:00 [queued]>
[2024-01-28T13:42:38.431+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T13:42:38.542+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-01-15 00:00:00+00:00
[2024-01-28T13:42:38.645+0000] {standard_task_runner.py:60} INFO - Started process 984 to run task
[2024-01-28T13:42:38.684+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-01-15T00:00:00+00:00', '--job-id', '997', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpoxpd7mkx']
[2024-01-28T13:42:38.703+0000] {standard_task_runner.py:88} INFO - Job 997: Subtask get_stackoverflow_data
[2024-01-28T13:42:39.150+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-01-15T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T13:42:40.049+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-01-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-15T00:00:00+00:00'
[2024-01-28T13:42:41.446+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T13:42:41.519+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230115T000000, start_date=20240128T134238, end_date=20240128T134241
[2024-01-28T13:42:42.443+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T13:42:42.739+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
